src/olympia/versions/admin.py

```diff
@@ -9,5 +10,30 @@ class LicenseAdmin(admin.ModelAdmin):
     ordering = ('builtin',)
 
 
+class VersionAdmin(admin.ModelAdmin):
+    view_on_site = False
+    readonly_fields = ('version', 'channel')
+
+    raw_id_fields = ('addon', 'license', 'release_notes')
```

Comment:
You shouldn't need to add `release_notes` to the raw id fields ? It's a translated field, it should be handled already.

Reply:
Done in 2b74fb06c3.